types/json-diff/index.d.ts

```diff
@@ -3,5 +3,9 @@
 // Definitions by: Tommy Wong <https://github.com/wchtommy20013/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function diff(obj1: any, obj2: any): any;
-export function diffString(obj1: any, obj2: any): any;
+export interface JsonDiffConfig {
+    color?: boolean;
+}
+
+export function diff(obj1: unknown, obj2: unknown): any;
+export function diffString(obj1: unknown, obj2: unknown, config?: JsonDiffConfig): string;
```

Comment:
```suggestion
export function diffString(obj1: unknown, obj2: unknown, config?: JsonDiffConfig): string;
```
so the 3rd is named `colorizeOptions`, while 4th `diffOptions`